package.json

```diff
@@ -15,7 +15,7 @@
     "@asyncapi/generator": "^1.0.0-rc.10",
     "@asyncapi/html-template": "^0.9.0",
     "@asyncapi/markdown-template": "^0.9.0",
-    "@asyncapi/parser": "^0.28.1",
+    "@asyncapi/parser": "^1.0.0-rc.3",
```

Comment:
please use latest rc.4

Reply:
Done. I also wasn't sure but do I need to check in the `package-lock.json` or omit it from PR?

Reply:
My recommendation is that you leave package-lock.json out of the PR. We'll handle it ourselves.

Reply:
Done, I've cherry-picked the commits that didnt include that change